cardano-node/src/Cardano/Tracing/Peer.hs

```diff
@@ -0,0 +1,203 @@
+{-# LANGUAGE ConstraintKinds       #-}
```

Comment:
You do not need `ConstraintKinds` in this code.

Reply:
Fixed.